### ISSUE TITLE
Threshold setting to separate `dayClick` from `select` callbacks

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2428]
Originally reported on Google Code with ID 2163

```
So that the `select` only happens after the mouse has been dragged a certain distance.

A pull request implementing this, via @treyhunner:
https://github.com/arshaw/fullcalendar/pull/140
```

Reported by `adamrshaw` on 2014-06-05 20:50:46

Imported with **3** stars.

**Comment by jbafford** [comment]
Do you want me to submit a PR for this branch? (I didn't before because the contribution guidelines discouraged doing that.)

**Comment by arshaw** [comment]
sure sounds good. can you link to this issue in the description?

**Comment by arshaw** [comment]
Released in [v3.2.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.2.0)

https://fullcalendar.io/docs/selection/selectMinDistance/
